src/qt/bitcoingui.cpp

```diff
@@ -1281,7 +1281,7 @@ void BitcoinGUI::updateProxyIcon()
 
 void BitcoinGUI::updateWindowTitle()
 {
-    QString window_title = "vBitcoin Core" ; //PACKAGE_NAME;
```

Comment:
If PACKAGE_NAME is Xenios SEZC
it will show that as the window title - which we don't want.
PACKAGE_NAME is generated by the makefile and put in 
bitcoin-config.h

Line 391: #define PACKAGE_NAME ("Xenios SEZC")

We do need to change the About Xenios SEZC page to add back in Bitcoin developers. But that was a phase 2 thing.